bundle-workflow/src/test_workflow/test_result/test_result_component.py

```diff
@@ -0,0 +1,23 @@
+from sortedcontainers import SortedDict  # type: ignore
+
+
+class TestResultsComponent(SortedDict):
```

Comment:
nit: rename TestComponentResults

Reply:
This one is my fault, we have some reverse naming in the project for checks. I don't have a strong preference one way or another.